spark-bigquery-pushdown/src/main/scala/com/google/cloud/spark/bigquery/BigQueryRelationUtil.scala

```diff
@@ -0,0 +1,46 @@
+package com.google.cloud.spark.bigquery
+
+import com.google.cloud.bigquery.{BigQuery, BigQueryOptions}
+import com.google.cloud.bigquery.connector.common.BigQueryProxyTransporterBuilder
+import com.google.cloud.http.HttpTransportOptions
+import org.apache.spark.internal.Logging
+
+import java.util.Optional
+
+object BigQueryRelationUtil extends Logging {
+
+  def createBigQuery(options: SparkBigQueryConfig): BigQuery = {
```

Comment:
This method should be part of the spark-bigquery-connector-common, and can use `BigQueryClientModule`

Reply:
Will make that change in the next commit

Reply:
Is there any reason we cannot pass the BigQueryClient created [here](https://github.com/GoogleCloudDataproc/spark-bigquery-connector/blob/master/spark-bigquery-dsv1/src/main/scala/com/google/cloud/spark/bigquery/BigQueryRelationProvider.scala#L76)  in the [BigQueryRelationProvider](https://github.com/GoogleCloudDataproc/spark-bigquery-connector/blob/master/spark-bigquery-dsv1/src/main/scala/com/google/cloud/spark/bigquery/BigQueryRelationProvider.scala#L76)?

Reply:
In the latest commit, I have passed the bigQueryClient into the relation. Let me know if I have missed something by doing that since it seemed the most straight forward. All the tests seemed to have passed too

Reply:
Sounds good